server/sandbox_run_linux.go

```diff
@@ -94,11 +94,11 @@ func (s *Server) configureSandboxIDMappings(mode string, sc *types.LinuxSandboxS
 	if uidMappingsPresent || gidMappingsPresent {
 		user := sc.RunAsUser
 		if user == nil {
-			return nil, errors.New("cannot use uidmapping or gidmapping if not running as root")
+			return nil, errors.New("cannot use uidmapping or gidmapping if not RunAsUser not set")
```

Comment:
I'll defer to @haircommander or @saschagrunert if this is right technically, but I think you want:
```suggestion
			return nil, errors.New("cannot use uidmapping or gidmapping if RunAsUser is not set")
```

Reply:
Thanks for point it out.  I'll push an update on Monday.

/hold